src/main/java/net/hypixel/nerdbot/curator/ForumChannelCurator.java

```diff
@@ -57,6 +57,10 @@ public List<GreenlitMessage> curate(ForumChannel forumChannel) {
                 Emoji agreeEmoji = getJDA().getEmojiById(NerdBotApp.getBot().getConfig().getEmojiConfig().getAgreeEmojiId());
                 Emoji disagreeEmoji = getJDA().getEmojiById(NerdBotApp.getBot().getConfig().getEmojiConfig().getDisagreeEmojiId());
                 DiscordUser discordUser = Util.getOrAddUserToCache(database, firstPost.getAuthor().getId());
+                if (discordUser == null) {
+                    continue;
```

Comment:
Should we not just break out of the loop instead of continuing the curator process if it's being done this early?

Reply:
Theoretically if only one discordUser is null, we aren't skipping the rest of the users